src/OmnichannelChatSDK.ts

```diff
@@ -2042,30 +2043,47 @@ class OmnichannelChatSDK {
                     this.scenarioMarker.completeScenario(TelemetryEvent.GetAuthToken);
                 } else {
                     const exceptionDetails = {
-                        response: "UndefinedAuthToken"
+                        response: ChatSDKErrorName.UndefinedAuthToken
                     };
 
+                    if (optionalParams?.throwError) {
+                        throw Error(exceptionDetails.response);
+                    }
```

Comment:
handle it on catch block, we shouldnt silent exceptioins

Reply:
The exception is only silenced on 1st attempt for backward compability. The 2nd attempt is performed in start chat. The exception would be thrown in start chat if the token is invalid